app/game_objects/weapon.py

```diff
@@ -0,0 +1,5 @@
+from app.game_objects.game_object import GameObject
+
+
+class Weapon(GameObject):

```

Comment:
Why did you inherit it from GameObject? Really weapon can have protection or hp?